src/clarity-angular/popover/signpost/signpost.specs.ts

```diff
@@ -11,385 +11,503 @@ import { Signpost } from "./signpost";
+                 * There are five things to test here
+                 * 1. Correct class name
+                 * 2. correct anchor point
+                 * 3. correct popover point
+                 * 4. Correct Y offset
+                 * 5. Correct X offset
+                 *
+                 */
+
+                signpost.signpostPosition = "left-top";
+                fixture.detectChanges();
+                let testClass = clarityElement.querySelector(".left-top");
+                expect(testClass).toBeDefined();
+
+                let leftTopPosition: TestPosition = {
+                    anchorPoint: Point.LEFT_CENTER,
+                    popoverPoint: Point.RIGHT_BOTTOM,
+                    offsetY: 0,
+                    offsetX: -14
+                };
+
+                testPosition(signpost, leftTopPosition);
+            });
         });
 
-        it("has a left-top signpost content position", function() {
-            /*********
-             *
-             * @description
-             * There are five things to test here
-             * 1. Correct class name
-             * 2. correct anchor point
-             * 3. correct popover point
-             * 4. Correct Y offset
-             * 5. Correct X offset
-             *
-             */
-
-            signpost.signpostPosition = "left-top";
-            fixture.detectChanges();
-            let testClass = clarityElement.querySelector(".left-top");
-            expect(testClass).toBeDefined();
-
-            expect(signpost.anchorPoint).toBe(Point.LEFT_TOP);
-            expect(signpost.popoverPoint).toBe(Point.RIGHT_BOTTOM);
-            expect(signpost.signpostOptions.offsetY).toBe(18);
-            expect(signpost.signpostOptions.offsetX).toBe(-14);
+        describe("custom trigger", function() {
+            let fixture: ComponentFixture<any>;
+            let clarityElement: any;
+            let signpost: Signpost;
+
+            beforeEach(() => {
+                TestBed.configureTestingModule({
+                    imports: [ ClrSignpostModule, ClrIconModule ],
+                    declarations: [ TestCustomTriggerSignpost ]
+                });
+
+                fixture = TestBed.createComponent(TestCustomTriggerSignpost);
+                fixture.detectChanges();
+                clarityElement = fixture.nativeElement;
+                signpost = fixture.componentInstance.signpost;
+
+            });
+
+            afterEach(() => {
+                fixture.destroy();
+            });
+
+            it("projects a custom trigger element to hide/show content", function() {
+                let signpostTrigger: HTMLElement =
+                    clarityElement.querySelector(".signpost-trigger > .signpost-action");
+                let signpostContent: HTMLElement;
+
+                expect(signpostTrigger.textContent.trim()).toBe("Custom trigger");
+
+                // Test we have a trigger
+                expect(signpostTrigger).not.toBeNull();
```

Comment:
Any way you could check that the default trigger is not also displayed? I could see this happening after a refactor or Angular breaking change and us not catching it.